lra/certifications/index.html.haml

```diff
@@ -0,0 +1,14 @@
+---
+layout: rightcol
+title: Community
+---
+
+%h1
+  LRA Certifications
+
+
+%h2{:class => page.output_path.start_with?("/lra") ? "active": ""}
+  %a{:href => relative("/lra/certifications/microprofile-lra-1.0/CCR-results/")} LRA 1.0 Compatible Certification Request (CCR)
```

Comment:
The href needs index.html adding (otherwise the link takes the user to directory listing.

Reply:
Also I think you have deleted the directory `CCR-results`

Reply:
basically, this works if you name the file CCR-results.html.haml then you can reference is in this way but I agree I'll change it to index.html

Reply:
thanks